tests/checkup_test.go

```diff
@@ -291,7 +291,7 @@ func newConfigMap() *corev1.ConfigMap {
 		"spec.timeout": testTimeout.String(),
 		"spec.param.networkAttachmentDefinitionName": networkAttachmentDefinitionName,
 		"spec.param.trafficGenPacketsPerSecond":      "8m",
-		"spec.param.testDuration":                    "1m",
+		"spec.param.testDuration":                    "4h",
```

Comment:
This change doesn't seem to be related to the PR.

Reply:
DONE